### PR TITLE
Loading wheel and Equalize outputs

### PR DESCRIPTION
**Pull request by olu144**
I have added a loading wheel that will be show on screen until charts have been fully loaded. This will ensure the user that they have submitted an input and deter them from attempting to submit another input. I have also tweaked server values to ensure that all input forms, channel id, video id, username and YouTube category, all output 10 comments. This will remove any discrepancies in loading times and provide for a more uniform user experience. 
Screenshot 2020-07-31 at 1 54 04 PM - Display 2 - Edited

### CODE REVIEW

**Review by danieldortega77 — APPROVED** [review]
Nice - LGTM

File: capstone/src/main/webapp/youtube_analyzer.html [hunk 1]
```diff
@@ -70,7 +70,8 @@ <h1>Youtube Analysis</h1>
         <button onclick="callYoutube()" class="btn btn-primary">Submit</button>
         <button onclick="getKeywordSearchResults()" class="btn btn-primary" id="keywordSearch">Keyword Search</button>
       </form><br>
-      <div id="comment-list"></div>
+       <div id="loading-container"></div>
+       <div id="comment-list"></div>
        <div id="chart-container"></div>
        <div id="table-container"></div>
```
Comment (chrismohrii) [inline capstone/src/main/webapp/youtube_analyzer.html]: nit: Do these lines have an extra indent? Also, is it possible to remove the `<br>` tag above with some CSS style changes?

**Review by chrismohrii — APPROVED** [review]

File: capstone/src/main/webapp/youtube_analyzer.html (hunk 1 above)
Comment (olu144) [reply capstone/src/main/webapp/youtube_analyzer.html]: addressed in "fix indenting and spacing"

**Review by dborkan — APPROVED** [review]
LGTM

File: capstone/src/main/webapp/youtubeTypescript.js [hunk 2]
```diff
@@ -577,3 +583,18 @@ function drawTableChart() {
     }
     table.draw(tableData, { allowHtml: true, showRowNumber: false, width: '100%', height: '100%' });
 }
+/** Displays a loading wheel that can be used a placeholder until an output is ready to be displayed*/
+function showLoadingWheel() {
+    var loadingContainerElement = document.getElementById('loading-container');
+    // Only one loading wheel will be shown at a time
+    if (loadingContainerElement.innerHTML == '') {
+        var loadingWheel = document.createElement('p');
```
Comment (dslucas) [inline capstone/src/main/webapp/youtubeTypescript.js]: nit: Let's use `const`s or `let`s where possible.

File: capstone/src/main/webapp/youtubeTypescript.js [hunk 3]
```diff
@@ -577,3 +583,18 @@ function drawTableChart() {
...
+        loadingWheel.className = 'spinner-border';
+        loadingContainerElement.appendChild(loadingWheel);
+    }
+}
+/** Removes the placeholding loading wheel*/
+function hideLoadingWheel() {
+    var loadingContainerElement = document.getElementById('loading-container');
```
Comment (dslucas) [inline capstone/src/main/webapp/youtubeTypescript.js]: Same here.

**Review by dslucas — APPROVED** [review]

File: capstone/src/main/webapp/youtubeTypescript.js (hunk 2 above)
Comment (olu144) [reply capstone/src/main/webapp/youtubeTypescript.js]: I have added youtubeTypescript.js to .gitignore in "add gitignore and delete cache"

**Pull request merged**